hack/quickstart-setup.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+#
+# Copyright 2021 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+if [ -z $KUADRANT_ORG ]; then
+  KUADRANT_ORG=${KUADRANT_ORG:="kuadrant"}
+fi
+if [ -z $KUADRANT_BRANCH ]; then
+  KUADRANT_BRANCH=${KUADRANT_BRANCH:="main"}
+fi
+if [ -z $MGC_BRANCH ]; then
```

Comment:
Do we want the branch you install from to be configurable? or do you mean it'd already be set at this stage before the curl request so no need to set a default?

I followed what was done here in MGC quickstart https://github.com/Kuadrant/multicluster-gateway-controller/blob/main/hack/quickstart-setup.sh#L19-L24

Reply:
right but I don't see `MGC_BRANCH` being used in the script. Perhaps I am missing something

Reply:
Ah I see, it will be in the MGC links once I've merged the other PR to main, I'll update the commands:
```bash
source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/${KUADRANT_ORG}/multicluster-gateway-controller/${MGC_BRANCH}/hack/.quickstartEnv)"
source /dev/stdin <<< "$(curl -s https://raw.githubusercontent.com/${KUADRANT_ORG}/multicluster-gateway-controller/${MGC_BRANCH}/hack/.deployUtils)"
```

Reply:
right ok. Hmm. I am wondering if perhaps we should split these pieces of work. I fear here we could end up with a situation where you need to know the MGC_BRANCH to use and the KUADRANT_BRANCH to use. As we could change something in MGC deploy utils and we might accidentally break this quick start. So I am thinking that it may keep us more sane and be in our best interest right now to copy over the scripts you need and add an issue to discuss how the projects can share these functions??

Reply:
So based on discussion we will do the following:
rely on mgc `main` scripts for 0.6.0 of kuadrant and then move to pin the scripts in 0.7.0